src/scripts/main.js

```diff
@@ -1,3 +1,209 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('tbody tr')];
+
+(function tableSorter() {
+  const categories = [...document.querySelectorAll(`thead th`)];
+  const backwards = [];
+
+  for (const categoryIndex in categories) {
```

Comment:
https://stackoverflow.com/questions/500504/why-is-using-for-in-for-array-iteration-a-bad-idea